cns/restserver/util.go

```diff
@@ -441,6 +499,45 @@ func (service *HTTPRestService) SendNCSnapShotPeriodically(ncSnapshotIntervalInM
 	}
 }
 
+// isNCWaitingForUpdate :- Determine whether NC version on NMA matches programmed version
+func isNCWaitingForUpdate(ncVersion, ncid string) (waitingForUpdate bool, returnCode int, message string) {
+	getNCVersionURL, ok := ncVersionURLs.Load(ncid)
```

Comment:
can you add log at start of function with ncid and ncversion

Reply:
either the caller will log or GetNetworkContainerVersion will